### DISCUSSION TITLE
Plotting two driver's race lap times into one scatter plot

### DISCUSSION THREAD

**Discussion by 3ric03** [theOehrly/Fast-F1#393] [Q&A]
Hi all,

I'm following the example on the fastf1 website where they plot the laptimes of a driver. 
https://docs.fastf1.dev/examples_gallery/plot_driver_laptimes.html#sphx-glr-examples-gallery-plot-driver-laptimes-py

I want to modify this to plot two driver laptimes into one scatter plot (effectively comparing them)
I have this code:

 ```
  merge_list = [lap_df_1, lap_df_2] #lap dataframe for driver 1 and driver 2
    merge_df = pd.concat(merge_list) #merge into one dataframe
    
    fig, ax = plt.subplots(figsize=(8, 8))
    print(type(merge_df))
    sns.scatterplot(data=merge_df,
                x="LapNumber",
                y="LapTime",
                ax=ax,
                linewidth=0,
                legend='auto')
```
but I'm getting the error, " raise ValueError("cannot reindex on an axis with duplicate labels")"

I understand that by combining two lap dataframe into one, I get duplicated indexes, but how can I solve this issue without deleting duplicates (which will erase the info I need)?

Thanks in advance for any help

**Comment by Casper-Guo** [comment]
You do not need to merge the dataframes. Just call the `sns.scatterplot` command once on each dataframe and the two scatterplots will be plotted onto the same canvas.

As in:

```
sns.scatterplot(data=lap_df_1...)
sns.scatterplot(data=lap_df_2...)
plt.show()
```
